packages/volto/src/components/manage/Widgets/FormFieldWrapper.jsx

```diff
@@ -22,6 +24,11 @@ const messages = defineMessages({
     id: 'Language independent field.',
     defaultMessage: 'Language independent field.',
   },
+  language_independent_icon_title: {
+    id: 'Language independent icon title',
+    defaultMessage:
+      'This field is a language-independent field. Any value you enter here will be synced between the translations when you save this form.',
```

Comment:
Description pulled from https://github.com/plone/plone.app.multilingual?tab=readme-ov-file#language-independent-fields

```suggestion
      'This is a language independent field. Any value you enter here will overwrite the corresponding field of all members of the translation group when you save this form.',
```

Reply:
Done.